news/2022-07-23-new-featured-artist-noisz.md

```diff
@@ -0,0 +1,67 @@
+---
+layout: post
+title: "New Featured Artist: NOISZ"
+date: 2022-07-20 10:00:00 +0000
+---
+
+**NOISZ** expands our Featured Artist library with a whole bunch of original rhythm game compositions by artists like *Soleily*, *Nhato* and *Camellia*!
+
+![](https://assets.ppy.sh/artists/286/header.jpg)
+
+An unprecedented combo of the bullet hell genre, visual novels, and (most importantly) rhythm games, **NOISZ** is a video game backed by premier talents in the modern rhythm game universe. 
+
+As of today, **NOISZ**'s awesome soundtrack is now part of osu!.
+
+[**NOISZ**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/286)houses **11** songs (and their instrumental versions) created by this outstanding roster of musicians:
+
+- 3M-Project
+- [Camellia](https://osu.ppy.sh/beatmaps/artists/31)
+- [HyuN](https://osu.ppy.sh/beatmaps/artists/33)
+- Iris
+- kors k
+- Nhato
+- [Reku Mochizuki](https://osu.ppy.sh/beatmaps/artists/163)
+- Soleily
+- Synthion
```

Comment:
```suggestion
- [Synthion](http://osu.ppy.sh/users/2872433)
```

not sure if this is worth adding to the article but yeah just in case this is Synthion's actual osu! profile

Reply:
probably out of scope/needs verification from our end